Trustroots/trustroots
# Update locale handling and selection UI

#### Proposed Changes

* Pull way more languages into spoken languages list
* Swap jquery based Angular select input to React version https://react-select.com/
* Generate Array version alongside Object version as a file, so that we don't have to generate it on the fly at the server

#### Testing Instructions

* Create user, modify their profile. Try adding and removing languages, saving in between.
   * You should find much more languages than before
   * List works better on mobile than previously
   * You should be able to save / remove languages just like before
 * Add hosting offer with the user
 * Create another user
   * Go to map and apply language search filter, use same and different languages as you saved for previous profile
   * You should see the previous user with right locale filter
   * The input should work in identical way to profile editing

A good small follow-up will be to apply more TR colored [theme](https://react-select.com/styles#overriding-the-theme) to this input but for now I just wanted to move this forward.



Resolves https://github.com/Trustroots/trustroots/issues/98
